src/backend/distributed/commands/table.c

```diff
@@ -4128,12 +4128,15 @@ ConvertNewTableIfNecessary(Node *createStmt)
 		Oid createdRelationId = RangeVarGetRelid(createTableAsStmt->into->rel,
 												 NoLock, missingOk);
 
+		/* not try to convert the table if it already exists and IF NOT EXISTS syntax is used */
+		if (createTableAsStmt->if_not_exists && IsCitusTable(createdRelationId))
+		{
+			return;
+		}
```

Comment:
let's move this under below check:

```c
if (ShouldCreateTenantSchemaTable(createdRelationId))
```

so that we don't accidentally introduce a functionality change for citus managed tables